test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -0,0 +1,269 @@
+   */
+  async hideToken(tokenName: string): Promise<void> {
+    console.log(`Hide token ${tokenName} on homepage`);
+    await this.driver.clickElement({ text: tokenName, tag: 'span' });
+    await this.driver.clickElement(this.assetOptionsButton);
+    await this.driver.clickElement(this.hideTokenButton);
+    await this.driver.waitForSelector(this.hideTokenConfirmationModalTitle);
+    await this.driver.clickElementAndWaitToDisappear(
+      this.hideTokenConfirmationButton,
+    );
+  }
+
+  async importTokenBySearch(tokenName: string) {
+    console.log(`Import token ${tokenName} on homepage by search`);
+    await this.driver.clickElement(this.tokenOptionsButton);
+    await this.driver.clickElement(this.importTokensButton);
+    await this.driver.waitForSelector(this.importTokenModalTitle);
+    await this.driver.fill(this.tokenSearchInput, tokenName);
+    await this.driver.clickElement({ text: tokenName, tag: 'p' });
+    await this.driver.clickElement(this.importTokensNextButton);
+    await this.driver.waitForSelector(this.confirmImportTokenMessage);
+    await this.driver.clickElementAndWaitToDisappear(
+      this.confirmImportTokenButton,
+    );
+  }
+
+  async openNetworksFilter(): Promise<void> {
+    console.log(`Opening the network filter`);
+    await this.driver.clickElement(this.networksToggle);
+    await this.driver.waitUntil(
+      async () => {
+        return await this.driver.findElement(this.allNetworksOption);
+      },
+      {
+        timeout: 5000,
+        interval: 100,
+      },
+    );
+  }
+
+  async selectNetworkFilterAllNetworks(): Promise<void> {
+    console.log(`Selecting "All networks" from the network filter`);
+    await this.driver.clickElement(this.allNetworksOption);
+    await this.driver.waitUntil(
+      async () => {
+        const label = await this.getNetworksFilterLabel();
+        return label === 'All networks';
+      },
+      { timeout: 5000, interval: 100 },
+    );
+  }
+
+  async selectNetworkFilterCurrentNetwork(): Promise<void> {
+    console.log(`Selecting "Current network" from the network filter`);
+    await this.driver.clickElement(this.currentNetworkOption);
+    await this.driver.waitUntil(
+      async () => {
+        const label = await this.getNetworksFilterLabel();
+        return label !== 'All networks';
+      },
+      { timeout: 5000, interval: 100 },
+    );
+  }
+
+  async waitUntilFilterLabelIs(label: string): Promise<void> {
+    console.log(`Waiting until the filter label is ${label}`);
+    await this.driver.waitUntil(
+      async () => {
+        const currentLabel = await this.getNetworksFilterLabel();
+        return currentLabel === label;
+      },
+      { timeout: 5000, interval: 100 },
+    );
+  }
+
+  async check_ifAssetIsVisible(assetName: string): Promise<boolean> {
+    const assets = await this.driver.findElements(this.tokenLisiItem);
+    for (const asset of assets) {
+      const text = await asset.getText();
```

Comment:
This is another anti-pattern we should avoid. The function check_ifAssetIsVisible is not used, so we are fine for now.

Reply:
This method is a copy-paste, i will take a note to delete this method in separate PR, we should not keep any dead code. Thanks for pointing out !